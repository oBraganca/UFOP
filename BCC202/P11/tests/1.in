1 jose 5 1 5 1990
1 guilherme 6 9 3 1998
1 valentina 9 1 3 1945
2
1 jesus 10 25 1 0000
3
