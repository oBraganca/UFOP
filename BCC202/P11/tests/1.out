09 | 01/03/1945 | valentina
10 | 25/01/0000 | jesus
06 | 09/03/1998 | guilherme
05 | 01/05/1990 | jose
